test/permit2/SetTokens/mintSetWithPermit2.t.sol

```diff
@@ -22,17 +22,18 @@ contract GaslessTest is Test, Permit2Utils, DeployPermit2 {
     using SafeTransferLib for ERC20;
     using SafeTransferLib for ISetToken;
 
-    bytes32 constant FULL_EXAMPLE_WITNESS_TYPEHASH = keccak256(
+    bytes32 constant WITNESS_TYPEHASH = keccak256(
         "PermitWitnessTransferFrom(TokenPermissions permitted,address spender,uint256 nonce,uint256 deadline,MintData witness)MintData(ISetToken _setToken,uint256 _amountSetToken,uint256 _maxAmountInputToken, bytes[] _componentQuotes,address _issuanceModule,bool _isDebtIssuance)TokenPermissions(address token,uint256 amount)"
     );
 
-    bytes32 public constant _TOKEN_PERMISSIONS_TYPEHASH = keccak256("TokenPermissions(address token,uint256 amount)");
+    bytes32 public constant _TOKEN_PERMISSIONS_TYPEHASH =
```

Comment:
Why the underscore prefix ?

Reply:
Copy pasted the value from Permit2 repo and that was the name of the variable, will remove the underscore to keep consistency